integration_test/ops_agent_test.go

```diff
@@ -215,6 +229,18 @@ func setupOpsAgent(ctx context.Context, logger *logging.DirectoryLogger, vm *gce
 	return setupOpsAgentFrom(ctx, logger, vm, config, locationFromEnvVars())
 }
 
+// restartOpsAgent restarts the Ops Agent and waits for it to become available.
+func restartOpsAgent(ctx context.Context, logger *logging.DirectoryLogger, vm *gce.VM) error {
+	if output, err := gce.RunRemotely(ctx, logger.ToMainLog(), vm, "", restartCommandForPlatform(vm.Platform)); err != nil {
+		logger.ToMainLog().Printf("stdout=%s\nstderr=%s", output.Stdout, output.Stderr)
```

Comment:
RunREmotely already prints stdout/stderr to the log, no need to do it manually. stdout/stderr is also included in err if there is an error.

Reply:
Oops, this was a leftover debugging thing, removed.